src/AppBridgeBlock.ts

```diff
@@ -172,6 +178,13 @@ export class AppBridgeBlock {
         return getColorsByColorPaletteId(colorPaletteId);
     }
 
+    public async createColorPalette(colorPaletteCreate: FrontifyColorPaletteCreate): Promise<FrontifyColorPalette> {
+        const colorPaletteApiCreate = mapFrontifyColorPaletteCreateToColorPaletteApiCreate(colorPaletteCreate);
+        colorPaletteApiCreate.project_id = this.getProjectId();
+
+        return createColorPalette(colorPaletteApiCreate as ColorPaletteApiCreate);
```

Comment:
I don't think we should do it like that. This seems to be pretty ugly. See my comment in the repository.
```suggestion
        const colorPaletteApiCreate = mapFrontifyColorPaletteCreateToColorPaletteApiCreate(
            colorPaletteCreate,
            this.getProjectId(),
        );

        return createColorPalette(colorPaletteApiCreate);
```